packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -0,0 +1,265 @@
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
+import { ApiReferencesTable, MEDIUM_GAP_SIZE, LARGE_GAP_SIZE } from './ApiReferencesTable';
+import { IApiProperty, IApiInterfaceProperty, IApiEnumProperty, IMethod, IApiReferencesTableSetProps } from './ApiReferencesTableSet.types';
+import { IEnumTableRowJson, ITableRowJson, ITableJson } from 'office-ui-fabric-react/lib/common/DocPage.types';
+import { PropertyType } from '../../utilities/parser/index';
+import { extractAnchorLink } from '../../utilities/extractAnchorLink';
+
+export interface IApiReferencesTableSetState {
+  properties: Array<IApiProperty>;
+  showSeeMore: boolean;
+}
+
+const apiReferencesTableTopMargin = mergeStyles({
+  marginTop: '40px'
+});
+
+const TITLE_LINE_HEIGHT = 31.5;
+
+export class ApiReferencesTableSet extends React.Component<IApiReferencesTableSetProps, IApiReferencesTableSetState> {
+  constructor(props: IApiReferencesTableSetProps) {
+    super(props);
+
+    this.state = {
+      properties: this._generatePropertyArray(),
+      showSeeMore: false
+    };
+  }
+
+  public render(): JSX.Element {
+    const { className } = this.props;
+    return (
+      <Stack gap={LARGE_GAP_SIZE} className={css(apiReferencesTableTopMargin, className)}>
+        {this._renderFirst()}
+        {this._renderEach()}
+      </Stack>
+    );
+  }
+
+  public componentDidMount(): void {
+    window.addEventListener('hashchange', this._onHashChange);
+  }
+
+  public componentWillUnmount(): void {
+    window.removeEventListener('hashchange', this._onHashChange);
+  }
+
+  public componentDidUpdate(prevProps: IApiReferencesTableSetProps, prevState: IApiReferencesTableSetState): void {
+    if (prevState.showSeeMore === false && this.state.showSeeMore === true) {
+      const hash = extractAnchorLink(window.location.hash);
+      const el = document.getElementById(hash);
+
+      if (el) {
+        // update scroll position
+        window.scrollTo({ top: el.offsetTop - TITLE_LINE_HEIGHT /* title */, behavior: 'smooth' });
+      }
+    }
+  }
+
+  private _renderFirst(): JSX.Element | undefined {
+    if (this.state.properties.length >= 1) {
+      const item = this.state.properties[0];
+      return this._renderReferencesTable(item);
+    }
+    return undefined;
+  }
+
+  private _renderEach(): JSX.Element | undefined {
+    if (this.state.properties.length > 1) {
+      return (
+        <Stack gap={MEDIUM_GAP_SIZE}>
+          <ActionButton
+            iconProps={{ iconName: this.state.showSeeMore ? 'SkypeCircleMinus' : 'CirclePlus' }}
+            onClick={this._onClickSeeMore}
+            onRenderText={this._onRenderText}
+            styles={{ root: { paddingLeft: '0px' }, textContainer: { paddingLeft: '4px' } }}
```

Comment:
Not that perf is a primary concern for documentation, but _I think_ in-lining these new Objects causes React to re-reconcile vs. if the objects were stored in some way.

Since these are likely only rendered once per page load, the above concern doesn't matter.